validation/process_rlimits.go

```diff
@@ -17,7 +17,12 @@ func main() {
 	if err != nil {
 		util.Fatal(err)
 	}
-	g.AddProcessRlimits("RLIMIT_NOFILE", 1024, 1024)
+	g.AddProcessRlimits("RLIMIT_AS", 1024, 1024)
+	g.AddProcessRlimits("RLIMIT_CORE", 1024, 1024)
+	g.AddProcessRlimits("RLIMIT_CPU", 1024, 1024)
+	g.AddProcessRlimits("RLIMIT_DATA", 1024, 1024)
+	g.AddProcessRlimits("RLIMIT_FSIZE", 1024, 1024)
+	g.AddProcessRlimits("RLIMIT_STACK", 1024, 1024)
```

Comment:
Could we use different values for each of the limits? So that it would check if the runtime does not mix them.
Ditto between the soft and hard value.

Reply:
updated.